nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-connection-configuration.js

```diff
@@ -1374,6 +1418,44 @@
             }
         },
 
+        /**
+         * Sorts the available prioritizers.
+         */
+        sortAvailablePrioritizers: function () {
+            var availablePrioritizersList = $('#prioritizer-available');
+            availablePrioritizersList.children('li')
+                .detach()
+                .sort(function (aElement, bElement) {
+                    var nameA = $(aElement).text();
+                    var nameB = $(bElement).text();
+                    return nameA.localeCompare(nameB);
+                })
+                .appendTo(availablePrioritizersList);
+        },
+
+        /**
+         * Adds the controls to the specified selected draggable element.
+         *
+         * @argument {jQuery} draggableElement
+         */
+        addControlsForSelectedPrioritizer: function (draggableElement) {
+            var removeIcon = $('<div class="draggable-control"><div class="fa fa-remove"></div></div>')
+                .on('click', function () {
+                    // remove the remove ice
```

Comment:
Minor typo. Appears to be from copying/pasting from `nf-parameter-contexts`. Can you please update both?